libs/src/services/web3Manager/Web3Manager.ts

```diff
@@ -116,6 +120,14 @@ export class Web3Manager {
     this.web3 = web3
   }
 
+  setDiscoveryProvider(discoveryProvider: DiscoveryProvider) {
+    this.discoveryProvider = discoveryProvider
+  }
+
+  useDiscoveryRelay() {
```

Comment:
does this get used anywhere?

Reply:
ah i meant to use this in the url switch, thanks!